spec/features/publish/courses/publishing_a_course_with_validation_errors_spec.rb

```diff
@@ -0,0 +1,197 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+feature 'Publishing courses errors', { can_edit_current_and_next_cycles: false } do
+  before do
+    Timecop.travel(Find::CycleTimetable.mid_cycle)
```

Comment:
You don't need to time travel anywhere

Reply:
Why not? What changed?

Is there not a period of time where providers cannot publish courses?

Reply:
No